mod/follow.php

```diff
@@ -60,7 +60,9 @@ function follow_content(App $a)
 	}
 
 	$uid = local_user();
-	$url = Strings::escapeTags(trim(defaults($_REQUEST, 'url', '')));
+
+	// Issue 4815: Silently removing a trailing @
```

Comment:
trailing = right
prefixing = left

Reply:
You and @AlfredSK are two nitpickers ;-)

Reply:
Haha. :-)